odata-csdl/13 Entity Container.md

```diff
@@ -446,6 +446,10 @@ that differ only in a type-cast segment, allowing to bind instances of
 different sub-types to different targets. If paths differ only in
 type-cast segments, the most specific path applies.
 
+Elsewhere in this specification, an [instance path](#PathExpressions) that starts
```

Comment:
Where? Please add a link.

Reply:
In several places, which all link back here. Shall I also add forward links?